visualdl/frontend/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "axios": "^0.16.1",
+    "express": "^4.16.2",
```

Comment:
This better put in the "devDependencies" use npm installl xxx --save-dev

Reply:
done